package.json

```diff
@@ -1,30 +1,45 @@
 {
+  "name": "remix-app",
   "private": true,
   "sideEffects": false,
   "type": "module",
   "scripts": {
-    "build": "remix build",
-    "dev": "remix dev",
+    "build": "remix vite:build",
+    "dev": "remix vite:dev",
+    "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
     "start": "netlify serve",
-    "typecheck": "tsc -b"
+    "typecheck": "tsc"
   },
   "dependencies": {
+    "@netlify/edge-functions": "^2.3.1",
+    "@netlify/functions": "^2.6.0",
+    "@netlify/remix-adapter": "^2.3.0",
+    "@netlify/remix-edge-adapter": "^3.2.0",
+    "@netlify/remix-runtime": "^2.2.0",
     "@remix-run/node": "*",
     "@remix-run/react": "*",
-    "cross-env": "^7.0.3",
+    "@remix-run/serve": "*",
```

Comment:
should those be more specific to versions with `vite` support?

Reply:
It's a magic value. When the Remix CLI installs a template it replaces the `*` version in any `@remix-run/*` packages with the latest version.